app/util/k8s/dcapt.tfvars

```diff
@@ -140,6 +146,12 @@ confluence_replica_count = 1
 # can be dataset restoration, resource requirements, number of replicas and others.
 confluence_installation_timeout = 20
 
+# Termination grace period
+# Under certain conditions, pods may be stuck in a Terminating state which forces shared-home pvc to be stuck
+# in Terminating too causing Terraform destroy error (timing out waiting for a deleted PVC).
+# Termination grace period is 0 by default. You can override it if for some reason you need a different value
+# confluence_termination_grace_period = 0
```

Comment:
As discussed let’s uncomment until the new TF release and create a follow-up task: comment back, retest and switch to TF version > 2.1.1